Update project initialization steps in quick-start.md file (#2365)
vuejs/docs: src/guide/quick-start.md

In this section we will introduce how to scaffold a Vue [Single Page Application](/guide/extras/ways-of-using-vue#single-page-application-spa) on your local machine. The created project will be using a build setup based on [Vite](https://vitejs.dev) and allow us to use Vue [Single-File Components](/guide/scaling-up/sfc) (SFCs).

> Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed, then run the following command in your command line with the current working directory as the folder in which you intend to create the project (without the `>` sign):

Reviewer: Let's rephrase a little bit to make the sentences shorter:

```suggestion
Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed and your current working directory is the one where you intend to create a project. Run the following command in your command line (without the `>` sign):
```
